src/main/java/graphql/GraphqlErrorBuilder.java

```diff
@@ -176,6 +184,28 @@ public Map<String, Object> getExtensions() {
         public String toString() {
             return message;
         }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) return true;
+            if (!(o instanceof GraphQLError)) return false;
+            GraphQLError that = (GraphQLError) o;
+            return Objects.equals(getMessage(), that.getMessage())
+                    && Objects.equals(getLocations(), that.getLocations())
+                    && Objects.equals(getErrorType(), that.getErrorType())
```

Comment:
`graphql.ErrorClassification` is an interface so this is a place where technically a bad `graphql.ErrorClassification` would break you

Reply:
I've now added this in the Javadoc - is there another way you would like this to be handled?

Reply:
No just articulating a challenge